linux_os/guide/services/base/service_smartd_disabled/rule.yml

```diff
@@ -11,19 +11,20 @@ description: |-
     {{{ describe_service_disable(service="smartd") }}}
 
 rationale: |-
-    SMART can help protect against denial of
-    service due to failing hardware. Nevertheless, if it is not needed or the
-    system's drives are not SMART-capable (such as solid state drives), it can be
-    disabled.
+    SMART can help ensure availability of systems by notifying system operators
+    of failing hardware. Nevertheless, if it is not needed or the
+    system's drives are not SMART-capable (such as solid state drives), this service
+    can be disabled.
 
-severity: unknown
+severity: low
 
 identifiers:
     cce@rhel6: 26853-2
     cce@rhel7: 80272-8
+    cce@rhel8: 82385-6
```

Comment:
@shawndwells please return this cce to the pool. Smartmontools is not a part of rhel8.